submissions/dummy_submission/the-nuclear-options-v1.0.md

```diff
@@ -0,0 +1,233 @@
+
+----------
+
+
+
+<!--Does it make sense to have an appendix? Should I just post this on reddit and link it instead?-->
+
+## Appendix: Cost comparison: 40MW of solar power and accumulators vs a 40MW nuclear starter setup
+
+For this comparision we assume 100% productivity for all machines.
+
+###  Solar setup cost
+
+40 000 kW / 42 kW = 953 Solar panels and 953 * 84% = 800 accumulators
+
+Total intermediates: 
+
+* 4000 batteries, 
+* 1600 iron plates
+* 4765 copper plates
+* 4765 steel plates
+* 14295 electric circuits
+
+Let us deconstruct the batteries:
+
+* 80k sulfuric acid
+* 1600 + (4000) = 5600 iron plates
+* 4765 + 4000 = 8765 copper plates
+* 4765 steel plates
+* 14295 electric circuits
+
+Let us also deconstruct the acid:
+
+* Water
+* 8000 sulfur
+* 5600 + (1600) = 7200 iron plates
+* 8765 copper plates
+* 4765 steel plates
+* 14295 electric circuits
+
+We now have basic materials in a comparable way.
+
+###  Nuclear setup cost
+
+40MW = 1 nuclear plant, 4 heat exchangers, and 7 steam turbines. However, also include some relevant infrastructure: 3 centrifuges, 30 mining drills, 10 storage tanks, 1000 pipes, 5 pumps.
+
+We can calculate the cost by using Factorio Lab by setting the list as items produced per minute:
+
+https://factoriolab.github.io/list?z=eJwrcDLVMox3CtMyiXc21TKPT9QyjjfQMjaId3HSMjSIdw7WMo13dgUyDQzUirW0nPLrnB3VyowBh1IODg__
+
+We get:
+
+* 800 advanced circuits
+* 800 concrete
+* 1690 electric circuits
+* 8270 copper wire
+* 5385 copper plates
+* 750 steel plates
+* 8745 iron plates
+* Thousands of uranium ore
+
+However this list involves duplicated counts because it not a simple list of independent materials. e.g. 43% of iron production is shown to go to steel production, and 19% to electric circuit production, thus these two portions can be deducted. Similar calcualtions show the following revision:
+
+*  Electric circuits (Discount 95% for advanced circuits)
+* Copper wire (Discount 61% for electric cirucits, 39% for advanced circuits)
+* Copper plates (Discount 77% for copper wire)
+* Iron plates (Discount 43% for steel plates, 19% for electric circuits)
+* By the same logic, we were able to eliminate 100% of iron ore, copper ore, coal, and crude oil from these calculations.
+
+We are left with:
+
+* 800 advanced circuits
+* 800 concrete
+* 660 electric circuits
+* 0 copper wire
+* 1239 copper plates
+* 750 steel plates
+* 3324 iron plates
+* Thousands of uranium ore
+
+Now lets deconstruct the advanced circuits so that we can make the resource costs comparable to the other scenario. Our final list is:
+
+We are left with:
+
+* 0 + (800 * 2 ) = 1600 plastic bars
+* 800 concrete
+* 660 + (800 * 2) = 1690 electric circuits
+* 0 copper wire
+* 1239 + (800 * 2) = 2839 copper plates
+* 750 steel plates
+* 3324 iron plates
+* Thousands of uranium ore
+
+
+
+Now, putting solar and nuclear side by side, in this respective order:

```

Comment:
the side by side comparison is not easy to follow IMO. how about a cost factor, as already used in a lot of other calculations and comparison methods?
In addition, you are considering only the individual construction cost for solar, but you are adding the running cost to the nuclear setup. This is not a proper comparison. If you consider the running cost of a nuclear setup by considering the uranium mining, then you must consider the chain to refine the uranium to sustain this 40MW. And then you need to calculate the power required for all of this, and do the same for the solar field.

Reply:
Hmmm, Firerazor is right. However, I would rather simply state in your comparison: "This is only initial construction costs", otherwise it gets fussy. Solar fields sometimes have radars and roboports, which also pull power (and lights?). And IMO the costs involved in maintaining nuclear is neglible relative to the power gained. So if you state you ignore these small accuracies, then it's good enough. I mean 80% difference is massive enough that the cost of your uranium miners.

Reply:
Given that I shall include also steam power and that fuel costs are huge there, I'll probably make two sections for the mathematics here, for setup and operation separately.

I will also recheck the power demand of fuel production: Perhaps I will compare 39MW of solar to 40MW of nuclear + 1MW for fuel production (although it increases very sublinearly for additional plants after kovarex).

For steam power, I wonder if I should include solid fuel and rocket fuel in boilers but I prefer not.

Reply:
agreed - no need to add even more fuel sources, it is already difficult enough :)